README.md

```diff
@@ -129,6 +129,6 @@ I would like to thank Microverse for their awesome template and my coding partne
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](/LICENSE) licensed.
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] As requested by the previous reviewer kindly fix the broken link to your MIT license agreement. Please note that you do not have a LICENSE.md file. Kindly add it